server/server.js

```diff
@@ -0,0 +1,104 @@
+import express  from "express";
+import cors from "cors";
+import DB from "./db.js";
+import bodyParser from "body-parser";
+import {v2 as cloudinary} from 'cloudinary';
+
+
+const app = express();
+const port = 3030;
+
+app.use(cors());
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+DB.connect();
+
+app.get("/", (req, res) => {
+    res.json("WELCOME");
+});
+
+app.get("/allcontacts", async (req, res) => {
+    try{
+        const result = await DB.query(`
```

Comment:
The project looks good. Question: Will you have additional get requests for your home page, or when looking up a single contact?